web-components/src/components/modal/ProcessingModal.tsx

```diff
@@ -27,25 +27,23 @@ const useStyles = makeStyles()(theme => ({
   },
 }));
 
-interface Props extends RegenModalProps {}
+export interface ProcessingModalProps extends RegenModalProps {}
```

Comment:
This type is used for the `processingModalAtom`. For now `ProcessingModalProps ` is the same as `RegenModalProps` but that might change in the future so I think it's best to wire the atom to this type instead of `RegenModalProps`.